plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/MockThriftHiveMetastoreHandler.java

```diff
@@ -0,0 +1,1699 @@
+    public String getMetaConf(String key)
+            throws MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public void setMetaConf(String key, String value)
+            throws MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public void createCatalog(CreateCatalogRequest catalog)
+            throws AlreadyExistsException, InvalidObjectException, MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public void alterCatalog(AlterCatalogRequest rqst)
+            throws NoSuchObjectException, InvalidOperationException, MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public GetCatalogResponse getCatalog(GetCatalogRequest catName)
+            throws NoSuchObjectException, MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public GetCatalogsResponse getCatalogs()
+            throws MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public void dropCatalog(DropCatalogRequest catName)
+            throws NoSuchObjectException, InvalidOperationException, MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public void createDatabase(Database database)
+            throws AlreadyExistsException, InvalidObjectException, MetaException, TException
+    {
+        delegate.createDatabase(database);
+    }
+
+    @Override
+    public Database getDatabase(String name)
+            throws NoSuchObjectException, MetaException, TException
+    {
+        Optional<Database> optionalDatabase = delegate.getDatabase(name);
+        return optionalDatabase.orElseThrow(() -> new NoSuchObjectException(""));
+    }
+
+    @Override
+    public void dropDatabase(String name, boolean deleteData, boolean cascade)
+            throws NoSuchObjectException, InvalidOperationException, MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public List<String> getDatabases(String pattern)
+            throws MetaException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
+    }
+
+    @Override
+    public List<String> getAllDatabases()
+            throws MetaException, TException
+    {
+        return delegate.getAllDatabases();
+    }
+
+    @Override
+    public void alterDatabase(String dbname, Database db)
+            throws MetaException, NoSuchObjectException, TException
+    {
+        throw new UnsupportedOperationException("method not supported");
```

Comment:
nit: Drop `"method not supported"`. The exception class is quite clear in this regard.